hyperspace/Makefile

```diff
@@ -26,8 +26,8 @@ build-release-hyperspace:
 	cargo b -p $(module-hyperspace) --release
 
 build-docker-image-hyperspace:
-	docker build -f scripts/hyperspace.Dockerfile -t "${composable_centauri_image_with_commit_hash}" .
-	docker tag "${composable_centauri_image_with_commit_hash}" "${composable_centauri_image_latest}"
+	docker build -f scripts/hyperspace.Dockerfile -t latest .
+	docker tag latest
```

Comment:
why did you remove the possibility of tracking a specific commit?

Reply:
sorry. fixing